ergo/platforms/metaculus/question/continuous.py

```diff
@@ -464,9 +464,9 @@ def show_prediction(
                         "For multiple predictions comparisons, only samples can be compared (plot_fitted must be False)"
                     )
                 for col in samples:
-                    df[col] = self.scale.normalize_points(samples[col])
+                    df[col] = onp.array(self.scale.normalize_points(samples[col]))
```

Comment:
@NixGD could you add a comment in the code here about what exactly this fixes?

(both for my benefit reviewing the PR, and for the benefit of future code maintainers)